db/db_impl/db_impl.h

```diff
@@ -2282,7 +2281,7 @@ class DBImpl : public DB {
 
   Directories directories_;
 
-  WriteBufferManager* write_buffer_manager_;
+  std::vector<std::shared_ptr<WriteBufferManager>> write_buffer_manager_;
```

Comment:
use autovector<Manager*> and put DB level manager separately from CF level managers.

Reply:
Good suggestion and done.